cirq/linalg/__init__.py

```diff
@@ -18,13 +18,8 @@
 Avoids duplicating functionality present in numpy.
 """
 
-from cirq.linalg.combinators import (
-    block_diag,
-    CONTROL_TAG,
-    dot,
-    kron,
-    kron_with_controls,
-)
+from cirq.linalg.combinators import (block_diag, CONTROL_TAG, dot, kron,
+                                     kron_with_controls, multi_dot)
```

Comment:
I think format made you do this because you don't have a trailing `,` after `multi_dot`?